surround/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pyaml==17.12.1
```

Comment:
`pyaml` is installed twice when running `poetry install`. Do we need both versions?

Reply:
Didn't realise that it's different from `pyyaml`. I wil have a look at this.